Code/Tools/AssetProcessor/native/ui/BuilderDataItem.cpp

```diff
@@ -0,0 +1,147 @@
+#include "BuilderDataItem.h"
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+namespace AssetProcessor
+{
+    const AZStd::string jobTypeDisplayNames[] = { "Analysis Jobs", "Processing Jobs" };
+    const AZStd::string invalidJobTypeDisplayName{ "Invalid Job Type" };
+
+    BuilderDataItem::BuilderDataItem(
+        ItemType itemType, const AZStd::string& name,
+        AZ::s64 jobCount,
+        AZ::s64 totalDuration,
+        AZStd::shared_ptr<BuilderDataItem> parent)
+        : m_itemType(itemType), m_name(name)
```

Comment:
you've put the rest of the initializers on their own lines, can you move m_name to its own?

Reply:
I will do that.

Reply:
resolved